client/src/components/Jobs/Jobs.scss

```diff
@@ -9,15 +9,15 @@
     }
     /* name */
     tr > *:nth-child(2) {
-      width: 30.5%;
+      width: 25%;
     }
     /* status */
     tr > *:nth-child(3) {
-      width: 21%;
+      width: 14em;
```

Comment:
the job status column needs to be a fixed width to accommodate the interactive session link buttons